[2025-08-15T03:36:29.811+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-15T03:36:29.865+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-15T03:35:44.816629+00:00 [queued]>
[2025-08-15T03:36:29.880+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-15T03:35:44.816629+00:00 [queued]>
[2025-08-15T03:36:29.883+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-15T03:36:29.906+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): validate_raw_reviews> on 2025-08-15 03:35:44.816629+00:00
[2025-08-15T03:36:29.926+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'playstore_review_etl_pipeline', 'validate_raw_reviews', 'manual__2025-08-15T03:35:44.816629+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/my_pipeline.py', '--cfg-path', '/tmp/tmpau7xcszf']
[2025-08-15T03:36:29.931+0000] {standard_task_runner.py:91} INFO - Job 137: Subtask validate_raw_reviews
[2025-08-15T03:36:29.931+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=194) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-15T03:36:29.933+0000] {standard_task_runner.py:63} INFO - Started process 196 to run task
[2025-08-15T03:36:30.015+0000] {task_command.py:426} INFO - Running <TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-15T03:35:44.816629+00:00 [running]> on host c54b9ccce26a
[2025-08-15T03:36:30.184+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@domain.com' AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='playstore_review_etl_pipeline' AIRFLOW_CTX_TASK_ID='validate_raw_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-08-15T03:35:44.816629+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-15T03:35:44.816629+00:00'
[2025-08-15T03:36:30.187+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-15T03:36:30.214+0000] {my_pipeline.py:107} INFO - Running validation on RawAppReviews data (Placeholder).
[2025-08-15T03:36:30.215+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-15T03:36:30.217+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-15T03:36:30.247+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=playstore_review_etl_pipeline, task_id=validate_raw_reviews, run_id=manual__2025-08-15T03:35:44.816629+00:00, execution_date=20250815T033544, start_date=20250815T033629, end_date=20250815T033630
[2025-08-15T03:36:30.350+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-15T03:36:30.452+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-15T03:36:30.460+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
